lib/reporting/reports/suppliers/helpers/columns_helper.rb

```diff
@@ -25,7 +25,7 @@ def producer_abn_acn
 
           def producer_charges_gst
             proc do |line_items|
-              supplier(line_items).charges_sales_tax ? I18n.t(:yes) : I18n.t(:no)
+              supplier(line_items).charges_sales_tax
```

Comment:
Given the specs are passing, I assume the boolean are auto-magically translated to Yes/No ?

Reply:
I was curious too, so I found out the answer here: https://github.com/openfoodfoundation/openfoodnetwork/blob/master/lib/reporting/report_row_builder.rb#L120-L122

I've been interested to try this gem for this sort of thing: https://github.com/RST-J/human_attribute_values
It would allow translators to choose a more specific description, which might be better in some circumstances. 

But I think handling it this way is quite sufficient and much simpler, until someone says we need a better solution.